src/libraries/Common/tests/System/Net/Http/HttpProtocolTests.cs

```diff
@@ -70,31 +69,22 @@ public async Task GetAsync_RequestVersion11_Success()
         [InlineData(0)]
         [InlineData(1)]
         [InlineData(9)]
-        public async Task GetAsync_RequestVersion0X_ThrowsOr11(int minorVersion)
+        public async Task GetAsync_RequestVersion0X_ThrowsNotSupportedException(int minorVersion)
         {
-            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            if (IsWinHttpHandler)
             {
-                using (HttpClient client = CreateHttpClient())
-                {
-                    HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, url);
-                    request.Version = new Version(0, minorVersion);
+                return;
+            }
 
-                    Task<HttpResponseMessage> getResponseTask = client.SendAsync(TestAsync, request);
-                    Task<List<string>> serverTask = server.AcceptConnectionSendResponseAndCloseAsync();
+            using (HttpClient client = CreateHttpClient())
+            {
+                HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://bing.com");
```

Comment:
I assume we fail before even trying to open a connection, and thus the URL doesn't actually matter and we don't need to make this outerloop?

Reply:
Yes.